capstone/src/main/java/com/google/sps/servlets/HardcodedClubs.java

```diff
@@ -0,0 +1,45 @@
+package com.google.sps.servlets;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+
+/* Creates hardcoded club information for prototype. */
+public class HardcodedClubs {
```

Comment:
Oh also, if this is only going to be used in servlets defined in this package, we can package scope this (and all of the constants) instead of making it public. Package scope means only classes in this package (eg: com.google.sps.servlets) can access it. It's somewhat between private (only this class can access) and public (any class anywhere can access). You just need to drop the public keyword to make it package-scoped.